test/dbtvault_harness_utils.py

```diff
@@ -114,15 +114,21 @@ def clean_target():
     shutil.rmtree(test.TEST_PROJECT_ROOT / 'target', ignore_errors=True)
 
 
-def clean_csv():
+def clean_csv(model_name=None):
     """
     Deletes csv files in csv folder.
     """
 
-    delete_files = [file for file in glob.glob(str(test.CSV_DIR / '*.csv'), recursive=True)]
+    if model_name:
+        delete_files = [test.CSV_DIR / f"{model_name.lower()}.csv"]
+    else:
+        delete_files = [file for file in glob.glob(str(test.CSV_DIR / '*.csv'), recursive=True)]
 
     for file in delete_files:
-        os.remove(file)
+        try:
```

Comment:
Avoid this. 

Use os.path.isfile() instead https://www.geeksforgeeks.org/python-os-path-isfile-method/

```
if os.path.isfile(file):
    os.remove(file)
```

Reply:
OK, I have revised the code and committed / pushed.